examples/tsl2561_simpletest.py

```diff
@@ -43,7 +43,10 @@
 print("Integration time = {}".format(tsl.integration_time))
 print("Broadband = {}".format(broadband))
 print("Infrared = {}".format(infrared))
-print("Lux = {}".format(lux))
+if lux != None:
+    print("Lux = {}".format(lux))
+else:
+    print("Lux = NaN")
```

Comment:
I'd rather not call it NaN because Python calls it None. I believe the format above will handle that ok.

Maybe we should add more info here too: https://github.com/adafruit/Adafruit_CircuitPython_TSL2561/blob/master/adafruit_tsl2561.py#L110

Reply:
if the added info turns into module documentation, then please, by all means. personally, i'm not expecting someone who purchases a TSL2561 BoB to read the module code but rather the module present important information in the example test code and in the specs (in the README, yes?)

Reply:
Ya, the comment in the example is helpful and the more info should be added to the function comment. It ends up here: https://circuitpython.readthedocs.io/projects/tsl2561/en/latest/api.html#adafruit_tsl2561.TSL2561.lux

I totally agree that some folks may not see it but having it more places, the better.

Lets just call it `None` for now but leave the `if` to show how to handle it. We can switch it to `nan` when the driver behavior changes.